core-util/FunctionPointer.h

```diff
@@ -83,10 +83,14 @@ class FunctionPointer0 : public FunctionPointerBase<R>{
     R call(){
         return FunctionPointerBase<R>::call(NULL);
     }
+    R operator ()(void) {
+        return FunctionPointerBase<R>::call(NULL);
+    }
 
     FunctionPointerBind<R> bind() {
-        FunctionPointerBind<R> fp;
-        fp.bind(&FunctionPointerBase<R>::_nullops, (ArgStruct *) NULL, this);
+        FunctionPointerBind<R> fp(*this);
+        void * storage = this->preBind(fp, (ArgStruct *)NULL, &FunctionPointerBase<R>::_nullops);
```

Comment:
@autopulated, I am not aware of any such agreement.
@bremoran:
- fluent APIs are going to look absolutely OK IMO
- internal only doesn't mean that we can use a different coding style